requirements/AUR_RT-0060/Taste-Leon3-Runtime-timers.txt

```diff
@@ -1,2 +1,6 @@
-Requirement fulfilled, document reference:
-    GR712RC 2018 Data sheet (2.4.6 General Purpose Timer Unit)
\ No newline at end of file
+TASTE Leon3 Runtime uses Leon3-BSP for implementing timers, as per [1]. 
```

Comment:
Chapter 5.1 does not mention Leon3 -BSP directly, right? Let's write "uses Timer module for accessing Leon3 hardware timers, as per  chapter 5.1 of [1]. Timer module is part of Leon3-BSP, as per chapter 5.2 of [1].  Leon3 hardware timers provide resolution of 50 ns, as per [2]". And modify [1] to not include chapter reference.

Reply:
Done